common/views/components/Body/Body.js

```diff
@@ -172,14 +174,49 @@ const Body = ({ body, isDropCapped, pageId }: Props) => {
             )}
 
             {slice.type === 'collectionVenue' && (
-              <VenueHours
-                venue={parseVenueTimesToOpeningHours(slice.value)}
-                isInList={
-                  pageId === 'openingTimes' || pageId === 'WwQHTSAAANBfDYXU'
-                }
-              />
+              <>
+                {slice.display && (
+                  <Layout8>
+                    <VenueClosedPeriods venue={slice.value} />
+                  </Layout8>
+                )}
+                {!slice.display && (
+                  <>
+                    <Layout
+                      gridSizes={
+                        pageId === 'openingTimes' ||
```

Comment:
This one (`pageId === 'openingTimes') can go now I think?

Reply:
What do you think about something like this to remove the conditional in the first place?
https://github.com/wellcometrust/wellcomecollection.org/pull/4405#discussion_r277585701

Reply:
I think that makes sense, will do that

Reply:
or have a slice for a collectionVenueList?

Reply:
What's the benefit of a list over adding multiple if you need it?
(not saying there isn't one, just unsure).

Reply:
Just thinking out loud really, but rather than having the layout determined by a label, it's whether it is part of a list of venues or a single venue that determines the layout

Reply:
So if there are multiple venues, it's wide, if not, i's not?
We already have the explicit logic of `featured`, which already does this with other components - Why not use that?

Reply:
I'll go with that